backend/tests/configuration/environment.test.ts

```diff
@@ -0,0 +1,29 @@
+import { assert, assertEquals, assertExists, assertThrows } from 'std/testing/asserts.ts';
+import { DotenvConfig } from 'https://deno.land/x/dotenv@v3.2.0/mod.ts';
+import { envVarsSchema } from '../../src/configuration/environment.ts';
+
+Deno.test('if config validation works', () => {
+  const config: DotenvConfig = {
+    PORT: '8000',
+    HOSTNAME: 'localhost',
+    FRONTEND_DIR: 'frontend',
+  };
```

Comment:
Part 1/2:
But this into a test independent const for the case, that we have to add more and more .env vars